cases/templates/cases/case_detail_staff.html

```diff
@@ -78,6 +78,36 @@ <h2 class="lbh-heading-h2 section-heading">Case info</h2>
       </a>
     </dd>
   </div>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Priority</dt>
+    <dd class="govuk-summary-list__value">
+    {% if case.priority %}
+        Yes
+    {% else %}
+        No
+    {% endif %}
+    </dd>
+    <dd class="govuk-summary-list__actions">
+        {% if case.priority %}
+            <a class="nw-link--no-visited-state" href={% url "case-priority-unset" case.id %}>
```

Comment:
Bit pedantic, but I think these should be buttons styled as links, and work on POST not GET, given that they change the state of the object. (You could then have a hidden set/unset field I guess with the one view, but that doesn't matter.)